### PR TITLE
add iota package from pypi

### PR DESCRIPTION
**Pull request by Anthchirp**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by alyubimov** [comment]
Happy to be a maintainer of this one.

**Comment by Anthchirp** [comment]
@conda-forge/help-python believe this is ready to go

File: recipes/iota/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - iota.filter_pickles = iota.command_line.iota_filter_pickles:entry_point
+    - iota.run = iota.command_line.iota_run:entry_point
+    - iota.single_image = iota.command_line.iota_single_image:entry_point
+    - iota.track_images = iota.command_line.iota_track_images:entry_point
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - pip
```
Comment (marcelotrevisani) [inline recipes/iota/meta.yaml:27]:
```suggestion
```

**Comment by marcelotrevisani** [comment]
If you want to, next time you can give a go to [Grayskull :skull: ](https://github.com/marcelotrevisani/grayskull/) to generate the recipes for packages on pypi for you

**Pull request merged**